app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -106,7 +106,7 @@ class Scene extends React.Component {
         wallWidth / 2,
         this.state.height / 2,
         wallWidth,
-        this.state.height,
+        this.state.height + 50,
```

Comment:
How'd you land on `50` here?

Reply:
I wanted the walls to have a height greater than the screen height, as the blocks are created at a height above the screen as well. They need to be at least a tad higher up than their creation height, as two can be created at the same-ish point and shove each other in a random direction. However regardless of what number I chose I felt like my height choice was a bit arbitrary so I just went with 50. I'm definitely open to suggestions on that though.

Reply:
Cool -- a comment there could help a future dev feel confident about changing that value in the future if need-be. You might also consider something like:

```javascript
// Padding applied to the height of the walls to prevent blocks created contemporaneously from colliding.
const padding = 50;
```